src/ert/gui/simulation/experiment_panel.py

```diff
@@ -297,3 +317,43 @@ def validationStatusChanged(self) -> None:
             self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
             and widget.isConfigurationValid()
         )
+
+    def populate_clipboard_debug_info(self) -> None:
+        kv = {"**Platform**": "", ":-----": ":-----"}
+        kv["Date"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        kv["OS"] = (
+            platform.system() + " " + platform.release() + " " + platform.machine()
+        )
+        kv["Hostname"] = platform.node()
+
+        for e in ["KOMODO_RELEASE"]:
+            if e in os.environ:
+                kv["Komodo release"] = os.environ[e]
```

Comment:
`os.environ.get("KOMODO_RELEASE", "")`?

Reply:
Yep, this was shrunk from a larger list. 🛠️